spec/index.md

```diff
@@ -2378,6 +2378,12 @@ The [standard nearest-rank estimation method](https://en.wikipedia.org/wiki/Perc
 
 ### IC method `bitcoin_get_block_headers` {#ic-bitcoin_get_block_headers}
 
+:::note
+
+The node metrics management canister API is considered EXPERIMENTAL. Canister developers must be aware that the API may evolve in a non-backward-compatible way.
```

Comment:
node metrics -> bitcoin get block headers

Reply:
Ha! Good catch, thanks!
I fixed it.